dataset-spec/README.md

```diff
@@ -0,0 +1,85 @@
+| url     | string | Organization homepage | Homepage of the provider.                       |
+
+###  Host Object
+
+| Element        | Type    | Name                  | Description                                                  |
+| -------------- | ------- | --------------------- | ------------------------------------------------------------ |
+| description    | string  | Description           | Detailed description to explain the hosting details. [CommonMark 0.28](http://commonmark.org/) syntax MAY be used for rich text representation. |
+| scheme         | string  | Scheme (required)     | Values: S3, GCS, URL, OTHER                                  |
+| id             | string  | Identifier (required) | Host-specific identifier such as an URL or asset id.         |
+| region         | string  | Region                | Provider specific region where the data is stored.           |
+| requester_pays | boolean | Requester pays        | `true` if requester pays, `false` if host pays.              |
+
+### Link Object
+
+TODO: Should be compatible with STAC or remove if catalog is revised.
+
+| Element | Type   | Name                           | Description                         |
+| ------- | ------ | ------------------------------ | ----------------------------------- |
+| href    | string | Hyperlink reference (required) |                                     |
+| rel     | string | Relation (required)            |                                     |
+| type    | string | MIME-type                      | MIME-type of the referenced entity. |
+| title   | string | Title                          | Human-readable title for the link.  |
+
+##  EO extension (eo) - Items and Datasets
+
+TODO
+
+We follow the STAC EO extension, but propose additional fields:
+
+| Element      | Type     | Name               | Description           |
+| ------------ | -------- | ------------------ | --------------------- |
+| unit         | ?        |                    |                       |
+| asset_schema | object   |                    |                       |
+| nodata       | [number] | Nodata values      | The no data value(s). |
+| pyramid      | ?        | Pyramid parameters |                       |
+| periodicity  | string   | Periodicity        | ISO8601               |
+
+#### Bands
+
+TODO
+
+We follow the STAC EO extension for bands, but propose additional fields:
+
+| Element   | Type     | Name          | Description                                                  |
+| --------- | -------- | ------------- | ------------------------------------------------------------ |
+| nodata    | [number] | Nodata values | The no data value(s).                                        |
+| data_type | string   | Data Type     | Data type for band values including its bit size. Values: uint8, uint16, uint32, uint64, int8, int16, int32, int64, float16, float32, float64 |
```

Comment:
I do think nodata, gain, and offset are useful to have per band, but not sure about data_type. The other fields may not be provided in the actual datafile (although providers should be encouraged to set them), but data_type will always be available.

Reply:
data_type was a field I ported from openEO and got spec'ed by my predecessor. I agree, I think it doesn't really make sense. Removed it for now.